src/tests/rest-calls-200-404.spec.ts

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import { AxiosHarTracker } from '../axios-har-tracker'
+import { writeFileSync } from 'fs';
+
+describe('Check axios-har-tracker for status 200 and 404', () => {
+
+    let axiosTracker, getHar
+
+    beforeAll(async () => {
+      axiosTracker = new AxiosHarTracker(axios);
+    });
+
+    afterAll(() => {
+      writeFileSync('example200404.har', JSON.stringify(getHar), 'utf-8')
+    });
+
+    it('should get har which includes status 200', async () => {
+      await axios.get('http://httpstat.us/200');
+      getHar = axiosTracker.getGeneratedHar();
```

Comment:
Rename getHar to generatedHar or something like that, variable names with verbs are usually used for functions, not static objects.